### DISCUSSION TITLE
[rejected] Uwzględnienie sugerowanego tytułu produktu w produktyzacji

### DISCUSSION THREAD

**Discussion by exploIF** [allegro/allegro-api#5477] [Suggestions - rejected]
Cześć,

Staram się dodać produkty do kategorii 10532, jednak sugerowana nazwa produktu nie jest brana pod uwagę, a ustalana na podstawie parametrów 237194, 207854 oraz 219781. W dokumentacji jest informacja o tym, że podawana nazwa produktu jest jedynie sugerowana, jednak bardzo zależy mi na tym aby została wzięta pod uwagę nazwa, którą wysyłam. Znajduje się w niej nazwa folii ochronnej, która różni się określonymi parametrami. Jesteśmy producentem produktu, po pierwszej próbie wysłania kilku produktów, wszystkie mają taką samą nazwę, ponieważ właśnie typ folii nie został wzięty pod uwagę, co jest mylące dla klientów. Istnieje jakaś możliwość, aby wymusić na Allegro wzięcie pod uwagę sugerowanej nazwy produktu albo dodać dodatkowy parametr w tej kategorii, który zostanie wzięty pod uwagę?

Pozdrawiam,
Igor

**Comment by Lukasz-Zurek** [comment]
Twój wątek ma charakter sugestii, dlatego przeniosłem go do zakładki "Discussions". Jednocześnie dziękuję za przesłane uwagi.

W naszym API nie ma takiej możliwości. Jeżeli jednak tytuły produktów nie spełniają Twoich oczekiwań, możesz zgłosić to jako błąd w produkcie. W tym celu skorzystaj z [POST /sale/products/{productId}/change-proposals](https://developer.allegro.pl/documentation/#operation/productChangeProposal), lub z formularza - [https://allegro.pl/offer/product/report/{productId}](https://allegro.pl/offer/product/report/%7BproductId%7D).

**Reply by hahyes** [reply]
> Jeżeli jednak tytuły produktów nie spełniają Twoich oczekiwań, możesz zgłosić to jako błąd w produkcie.

Forum Gadane jest pełne przypadków, gdzie zgłoszenia generycznego tytułu są odrzucane z góry. Obawiam się, że nie jest to rozwiązanie problemu.
